test/system/test_tpm2_certify.sh

```diff
@@ -38,8 +38,9 @@ file_certify_key_name=name.load.B1_B8
 file_output_attest=attest.out
 file_output_signature=certify_signature.out 
   
-
+# do not update alg_hash without updating alg_hash_name
 alg_hash=0x000B
+alg_hash_name=sha256
```

Comment:
I wanted to mix and match the hex values and the nice names to get coverage on both paths.